src/js/edu-benefits/pages/applicantInformation.js

```diff
@@ -0,0 +1,59 @@
+import _ from 'lodash/fp';
+
+import * as currentOrPastDate from '../../common/schemaform/definitions/currentOrPastDate';
+import * as fullName from '../../common/schemaform/definitions/fullName';
+import * as ssn from '../../common/schemaform/definitions/ssn';
+
+import { relationshipLabels, genderLabels } from '../utils/helpers';
+
+const defaultProps = [
+  'relativeFullName',
+  'relativeSocialSecurityNumber',
+  'relativeDateOfBirth',
+  'gender',
+  'relationship'
+];
+
+export default function applicantInformation(
+  schema,
+  fields = defaultProps,
+  required = ['relativeFullName']) {
+  const possibleProperties = {
+    relativeFullName: fullName.schema,
+    relativeSocialSecurityNumber: ssn.schema,
+    relativeDateOfBirth: currentOrPastDate.schema,
```

Comment:
Oh, I remember. It was `toursOfDuty` that was inconsistent. In 1995, it's fleshed out in the property, but in 5490, the property is a `$ref` to the definition...which is exactly the same.

At any rate, I'll pull them out!